File: python/program/transmisi.py
# transmisi.py
class Transmisi:
    # constructor
    def __init__(self, jenis_transmisi=""):
        self.jenis_transmisi = jenis_transmisi

    # Setter
    def set_jenis_transmisi(self, jenis_transmisi):
        self.jenis_transmisi = jenis_transmisi

    # Getter
    def get_jenis_transmisi(self):
        return self.jenis_transmisi

File: java/program/Transmisi.java
// Transmisi.java
public class Transmisi {
    // atribut
    private String jenisTransmisi;

    // Constructor default
    public Transmisi() {}

    // Constructor dengan parameter
    public Transmisi(String jenisTransmisi) {
        this.jenisTransmisi = jenisTransmisi;
    }

    // Setter
    public void setJenisTransmisi(String jenisTransmisi) {
        this.jenisTransmisi = jenisTransmisi;
    }

    // Getter
    public String getJenisTransmisi() {
        return jenisTransmisi;
    }
}
